cumulus_library/study_parser.py

```diff
@@ -248,10 +248,7 @@ def clean_study(
         # study builder, and remove them from the list.
         for view_table in view_table_list.copy():
             if any(
-                (
-                    (f"_{word.value}_") in view_table[0]
-                    or view_table[0].endswith(word.value)
-                )
+                f"__{word.value}_" in view_table[0]
```

Comment:
Only because checking for whether the table ended with the reserved word was explicitly against what the docs said was OK - and is different than the actual enforcement check in the code.